src/pudl_rmi/formatter_optimus.py

```diff
@@ -113,7 +115,9 @@
     "plant_id_eia": "Plant ID EIA",
     "report_year": "Report Year",
     "data_source": "Data Source of Depreciation Study",
-    "ferc_acct_name": "FERC Acct",
+    "ferc_acct_name": "FERC account",
+    "ferc_account_name_rmi": "technology",
+    "unaccrued_balance_w_common": "Unaccrued Balance",
```

Comment:
these are bb changes in the output bc of a change in what rmi needed (you can more or less ignore these)